tests/run.bats

```diff
@@ -33,6 +33,31 @@ load '../lib/run'
   unstub buildkite-agent
 }
 
+@test "Run without a prebuilt image with a complicated command" {
+  export BUILDKITE_JOB_ID=1111
+  export BUILDKITE_PLUGIN_DOCKER_COMPOSE_RUN=myservice
+  export BUILDKITE_PIPELINE_SLUG=test
+  export BUILDKITE_BUILD_NUMBER=1
+  export BUILDKITE_COMMAND="sh -c 'echo hello world'"
+  export BUILDKITE_PLUGIN_DOCKER_COMPOSE_CHECK_LINKED_CONTAINERS=false
+  export BUILDKITE_PLUGIN_DOCKER_COMPOSE_CLEANUP=false
+
+  stub docker-compose \
+    "-f docker-compose.yml -p buildkite1111 build --pull myservice : echo built myservice" \
+    "-f docker-compose.yml -p buildkite1111 run --name buildkite1111_myservice_build_1 myservice $BUILDKITE_COMMAND : echo ran myservice"
```

Comment:
Maybe it's worth copy+pasting the actual command into the expectations, rather than using $ substitution, only because the test is related to $ substitution?

Reply:
Yeah 🤔 I kept running into copy-paste errors was why I did that! 😅

Reply:
😂